File: design-parking-system/ParkingSystem.java
class ParkingSystem {

    public ParkingSystem(int big, int medium, int small) {
        
    }
    
    public boolean addCar(int carType) {
        
        return true;
        
    }
    
    
    public static void main(String[] args) {
        
        ParkingSystem parkingSystem = new ParkingSystem(1,1,0);
        
        System.out.println("Pass test 1: " + String.valueOf(parkingSystem.addCar(1) == true));
        System.out.println("Pass test 2: " + String.valueOf(parkingSystem.addCar(2) == true));
        System.out.println("Pass test 3: " + String.valueOf(parkingSystem.addCar(3) == false));
        System.out.println("Pass test 4: " + String.valueOf(parkingSystem.addCar(1) == false));
        
    }
}

/**
 * Your ParkingSystem object will be instantiated and called as such:
 * ParkingSystem obj = new ParkingSystem(big, medium, small);
 * boolean param_1 = obj.addCar(carType);
 */

File: design-parking-system/csharp/ParkingSystem/ParkingSystem.cs
// Hello World! program
namespace ParkingSystem
{
    class ParkingSystem
    {
        private int big;
        private int medium;
        private int small;

        public ParkingSystem(int big, int medium, int small)
        {
            this.big = big;
            this.medium = medium;
            this.small = small;
        }

        private bool AddCar(int v)
        {
            return true;
        }

        static void Main(string[] args)
        {

            ParkingSystem parkingSystem = new ParkingSystem(1, 1, 0);

            System.Console.WriteLine("Pass test 1: " + (parkingSystem.AddCar(1) == true));
            System.Console.WriteLine("Pass test 2: " + (parkingSystem.AddCar(2) == true));
            System.Console.WriteLine("Pass test 3: " + (parkingSystem.AddCar(3) == false));
            System.Console.WriteLine("Pass test 4: " + (parkingSystem.AddCar(1) == false));

        }


    }
}